regparser/tree/struct.py

```diff
@@ -51,6 +52,10 @@ def depth(self):
         else:
             return len(self.label)
 
+    @staticmethod
+    def is_markerless_label(label):
+        return re.match(r'p\d+', label[-1])
```

Comment:
@cmc333333 mind if we open an issue for the `regparser.tree.depth.markers.MARKERLESS` issue?